packages/dd-trace/src/profiling/profilers/wall.js

```diff
@@ -69,8 +69,12 @@ function ensureChannelsActivated () {
 }
 
 class NativeWallProfiler {
+  type = 'wall'
+  _mapper
+  _pprof
```

Comment:
note to self: do not use field = undefined, otherwise it ends up being a ugly lone label like that